### PR TITLE
Tails 4 Screenshots and docs updates

### PR DESCRIPTION
**Pull request by rocodes**
* Update screenshots in Disk Utility and Network Manager
* Update docs to refer to KeePassXC (and compatibility note), slight modifications where Tails 4 UI has changed
* Remove reference to "Seahorse" (just called Passwords and Keys)

## Status

Ready for review

## Description of Changes

Changes proposed in this pull request: docs only

## Testing

Manual review

### If you made changes to documentation:

- [x] Doc linting (`make docs-lint`) passed locally

### CODE REVIEW

File: docs/set_up_admin_tails.rst [hunk 1]
```diff
@@ -131,8 +131,8 @@ signed with the release signing key:
 .. code:: sh
 
     cd ~/Persistent/securedrop/
-    git checkout 1.0.0
```
[the remark this answers is not in the document]
Comment (rocodes) [reply docs/set_up_admin_tails.rst]: ah gotcha! thank you

File: docs/set_up_admin_tails.rst [hunk 2]
```diff
@@ -157,9 +157,14 @@ using this template.
 You can find the template in ``tails_files/securedrop-keepassx.kdbx``
```
[the remark this answers is not in the document]
Comment (rocodes) [reply docs/set_up_admin_tails.rst]: thanks! fixed

**Comment by rocodes** [comment]
> Thanks @rocodes the updates look great. The following images have 0755 (other images are 0644):
> 
> * docs/images/firewall/IPv4_settings.png
> * docs/images/firewall/tails_network_settings.png
> * docs/images/firewall/three_nic_admin_workstation_static_ip_configuration.png
> * docs/images/screenshots/create-passphrase.png
> 
> Could you please drop the executable bit on those? I've also provided a couple of comments inline for discussion.

Fixed file perms- thank you for catching - as well as including other fixes

**Comment by redshiftzero** [comment]
merging since the Tails 4 release day is today

**Comment by eloquence** [comment]
For cross-reference, this PR is towards resolving #4893.

**Pull request merged**